<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="branchBtn">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>支路信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clever</string>
    </property>
   </widget>
   <widget class="QToolButton" name="setBtn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>参数设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="alarmBtn">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>20</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>当前报警</string>
    </property>
   </widget>
   <widget class="QToolButton" name="logBtn">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>数据纪录</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLab">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IDC-BusBar</string>
    </property>
   </widget>
   <widget class="QLabel" name="ratedLab">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>260A</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWid">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>1021</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QToolButton" name="homeBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>监控主界面</string>
    </property>
   </widget>
   <widget class="QToolButton" name="lineBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>530</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>主路信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLab">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>2017-06-30 </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>540</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>01 主路源</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02 主路源</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03 主路源</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04 主路源</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="busNameLab">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>bus</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
